website/routes.py

```diff
@@ -262,6 +262,12 @@ def make_url_map(app):
             json_renderer
         ),
         Rule(
+            '/help/',
+            'get',
+            website_views.redirect_help_to_faq,
```

Comment:
I agree with @mfraezz 's suggestion. See this recent, relevant change: https://github.com/CenterForOpenScience/osf.io/pull/5555/files#diff-758f58f82207e3674b3105d2b3d4ce7aR215 . You should just be able to add `/help/` redirect to the Guides pages. @dsb9ef Can you please send a new PR branched off of https://github.com/CenterForOpenScience/osf.io/tree/caseyrollins-feature/getting-started-to-help?